project-progression-policy.md

```diff
@@ -137,6 +137,61 @@ To be considered for the Sandbox Stage, the project must meet the following requ
 
 * Upon acceptance, Sandbox projects must list their status prominently on their website/README
 
+#### Sandbox Benefits
+
+**Advantages of Sandbox vs non-Sandbox for new projects**
+
+CCC will remain fair and open to all projects no matter what their initial provenance.  Should a project apply for CCC Incubation, the TAC will use the same criteria regardless of origin. This means the TAC will not discriminate in favour of Sandbox projects vs. non-Sandbox projects.
+
+Therefore the advantages of being in the Sandbox apply prior to application for Incubation, includes:
+
+1. That a project has a legally neutral home that is stable and known
+
+2. And that a project may attain Incubation-level success faster:
+
+    1. Due to public visibility and association with the CCC mission
+    2. Through alignment with other CCC projects (if and only if desired)
+    3. Via other CCC and Linux Foundation services etc.
+
+3. The CCC will help projects adopt good principles of governance
+
+**Neutral Home**
+
+A neutral home for your project increases the willingness of developers from other companies and independent developers to collaborate, contribute, and become committers. Neutrality requires that projects contribute their trademark to the CCC so that:
+
+* No company is favored over any other
+* CCC ensures project governance is transparent and fair for everyone.
+
+**Clarifying Marketing Expectations**
+
+All open source projects in some sense enjoy a level of promotion from community, user enthusiasm, sponsoring organisations and so on. Please note that in this section we discuss marketing as a measurable financial investment into CCC projects from the CCC outreach budget and staff.
+
+Since the Sandbox is for early stage, sandbox projects will receive minimal marketing support from the foundation.  The Sandbox group as a whole may be promoted from time to time, which includes but is not limited to the following activities:
+
+* Developer community support: hangouts, meetups, events and (some) conferences
+* Digital marketing: help with online content, interactive tutorials, webinars, and social
+* Communnity marketing: conference promotion, landscape, certification, case studies, analyst relations, press relations
```

Comment:
Can you clarify what you anticipate the CCC doing for this list?
`conference promotion, landscape, certification, case studies, analyst relations, press relations`
**Conference Promotion**: We can promote conferences - no question there.
If we create a landscape view of projects a sandbox project would be included in it. Did you mean something else?
**Certification**: This sounds like future scope that we delegated to the GRC SIG. Might be premature to list here. 
**Case studies**: Typically a project would produce their own content here and then the CCC can promote it. I wouldn't want to communicate that the CCC would fund creating a case study unless we already have that budgeted somewhere?
**Analyst and Press Relations**: We do these things at the CCC level.